docs/Design/Exceptions/Exception Handling.md

```diff
@@ -0,0 +1,286 @@
+# Exception Handling
+
+This doc will cover both try/catch/finally logic.
+Try blocks contain code that might throw exceptions.
+Catch blocks detect these exceptions, and run the code contained if the exception is present.
+Finally blocks always run.
+
+## Commands
+
+### `try start`
+
+`try start`
+
+Begins a try block.
+
+### `try end`
+
+`try end`
+
+Ends a try block.
+
+### `catch start`
+
+`catch start : exceptionType *alias`
```

Comment:
Exception classes are (in the vast majority of languages, and at least the ones GLS currently supports) typically PascalCase and not camelCase. You should reflect that here.